test/Profiler/coverage_num_threads.swift

```diff
@@ -0,0 +1,14 @@
+// RUN: %target-swift-frontend -profile-generate -profile-coverage-mapping -num-threads 0 -emit-ir %S/Inputs/coverage_num_threads1.swift | %FileCheck %s -check-prefix=SINGLE-SOURCE
+
+// SINGLE-SOURCE: __LLVM_COV,__llvm_covmap
+// SINGLE-SOURCE-NOT: __LLVM_COV,__llvm_covmap
+
+// RUN: %target-swift-frontend -profile-generate -profile-coverage-mapping -num-threads 0 -emit-ir %S/Inputs/coverage_num_threads1.swift %S/Inputs/coverage_num_threads2.swift | %FileCheck %s -check-prefix=SINGLE-OBJECT
+
+// SINGLE-OBJECT: __LLVM_COV,__llvm_covmap
+// SINGLE-OBJECT-NOT: __LLVM_COV,__llvm_covmap
+
+// RUN: %target-swift-frontend -profile-generate -profile-coverage-mapping -num-threads 12 -emit-ir %S/Inputs/coverage_num_threads1.swift %S/Inputs/coverage_num_threads2.swift | %FileCheck %s -check-prefix=MULTIPLE-OBJECTS
```

Comment:
Interesting, I'm not sure how modules get split up for parallel compilation. Is -num-threads=12 necessary here, or can a smaller number work?

Reply:
any multi-threaded number should work, changed to 2